library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -325,7 +369,20 @@ PlatformBridgeFilter::encodeTrailers(Http::ResponseTrailerMap& trailers) {
   return status;
 }
 
+void PlatformBridgeFilter::resumeDecoding() {
+  auto weak_self = weak_from_this();
```

Comment:
nit: usually `weak_this` in the Envoy codebase

Reply:
I referenced the cache filter for this, and it used "self". (shrug emoji)

I have no preference.